### DISCUSSION TITLE
[1723260]真探(本体)

### DISCUSSION THREAD

**Discussion by 1271620983** [pjy612/SteamManifestCache#415] [Share]
`appid:1723260`
keys:
```
1723261;0ceea98d91910a55afb72d40979de4ca015d102c3837f0e6cb7eee5d312fab56
```
[[1723260]真探.zip](https://github.com/pjy612/SteamManifestCache/files/14019717/1723260.zip)

code:F57F96B8848ACBF4
email:hsj@hsj-home.online

**Comment by pjy612** [comment]
看更新说明
